lib/pluggy_elixir/auth.ex

```diff
@@ -0,0 +1,51 @@
+defmodule PluggyElixir.Auth do
+  @moduledoc """
+  Handle authentication actions.
+  """
+
+  alias PluggyElixir.Config
+  alias PluggyElixir.Error
+
+  @type t :: %__MODULE__{
+          api_key: binary()
+        }
+
+  defstruct [:api_key]
```

Comment:
According to [Elixir Style Guide](https://github.com/christopheradams/elixir_style_guide#module-attribute-ordering) `defstruct` should come after `module attributes`.